package.json

```diff
@@ -50,11 +50,11 @@
     "accessible-autocomplete": "^2.0.3",
     "applicationinsights": "^1.8.8",
     "awilix": "^8.0.0",
-    "axios": "^1.2.1",
+    "axios": "^0.21.4",
     "axios-oauth-client": "^1.4.4",
     "axios-token-interceptor": "^0.2.0",
     "codeceptjs": "^3.3.7",
-    "config": "3.3.7",
+    "config": "^3.3.2",
```

Comment:
The two dependencies in here need changing back to the master ones (1.21 and 3.37)

Reply:
Fixed